cmd/config.go

```diff
@@ -176,6 +179,10 @@ func (config *Config) Print() {
 		glog.Infof("Postgres URI: %s", config.PostgresURI)
 	}
 
+	if config.PosValidatorSeedHex != "" {
```

Comment:
Hmm seed hex not mnemonic? I guess because (mnemonic)->seedHex is a one-way operation, so everyone will have the latter but NOT everyone will have the former right?

Reply:
AFAIK, there isn't a standard derivation path to convert a BIP39 seed phrase to BLS private key. So we use the standard seedHex -> bls.PrivKey conversion provided by the FlowCrypto package.

Reply:
Hmm... so BIP39 is actually supposed to go from `mnemonic -> seedHex`, not to be confused with bip32 which deals with derivation paths and things like that. You can open identity and hit "advanced" on the part where it shows you a seed phrase and then see how it converts it:
https://ibb.co/wwW1rKH

My thinking is it would probably be good to make it so that the flag is a mnemonic, and that the seed hex used is identical to that generated by identity, which is standard bip39 generation.

I could also see the argument for keeping it as raw seed hex. In that case we'll just have to tell people to copy it from the advanced section or something. Also it feels easier to lose a seed hex than a mnemonic but idk.

Reply:
There should be a function somewhere that's like MnemonicToSeed or SeedToMnemonic

Reply:
Oh, I like this. And agreed that a seed phrase is easier to manage for node operators, vs a more arbitrary seed hex. The one thing I'm not sure of is whether the seed hex generated via BIP39 matches the number of bytes that the bls private key expects. I'll try to update to this.

Reply:
Implemented in PR https://github.com/deso-protocol/core/pull/946